src/site/content/en/blog/floc/index.md

```diff
@@ -26,6 +26,21 @@ cohort", which will be the same for thousands of browsers with a similar recent
 The browser recalculates its cohort periodically, on the user's device, without sharing
 individual browsing data with the browser vendor or anyone else.
 
+{% Aside %}
+A page visit will only be included in the browser's FLoC calculation if `document.interestCohort()` 
```

Comment:
I think this implies some certainty for how things will work after the origin trial. Probably best to just keep this to the inclusion criteria for the origin trial as anything else is still to be decided.

Reply:
Fair point — updated.